pkg/skaffold/test/custom/custom_test.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package custom
+
+import (
+	"context"
+	"io/ioutil"
+	"runtime"
+	"testing"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner/runcontext"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
+	"github.com/GoogleContainerTools/skaffold/testutil"
+)
+
+func TestNewCustomTestRunner(t *testing.T) {
```

Comment:
Can we add some tests for `TestDependencies`
https://github.com/GoogleContainerTools/skaffold/blob/dafb4152be74c7cd7d363790f2099f2b6f057ac3/pkg/skaffold/build/custom/dependencies_test.go

Reply:
Added unit tests for TestDependencies